### ISSUE TITLE
about normalization and sliding windows

### ISSUE THREAD

**Issue by Guan-t7** [thuml/Autoformer#27]
Hi, I'd like to confirm and discuss with you the following details of your experiment:

1. Z-score normalization is applied for each series, whose params are estimated from training part of individual series. Model predictions are **NOT** inverse-transformed to the original scale for metric compute. i.e. the MAE and MSE metrics presented in the paper are computed from normalized data. //
That's not very intuitive, as the result somehow depends on the preprocessing. I've read https://github.com/thuml/Autoformer/issues/6. This configuration seems to be inherited from previous baselines. Is there any rationale for this? I know that if you scale across the entire training set (a scalar \mu and \sigma), you can obtain MAE/MSE of original scale with a constant multiplier. But that's not the case here.

2. The time window is slided by 1 for next sample. i.e. there's only one new step of info between consecutive samples. //
I think we can more quickly cover the entire set if sliding by 1 < s <= `pred_len`. I wonder if it's something stated, or just a small detail that is really up to me?

Sorry for possible silly questions (they're not obvious for me and from the paper

**Comment by Guan-t7** [comment]
Wow, thank you for your prompt reply. That's very clear now.
